dev3/2024-07-02/example-metadata/min-specimen-biosample.json

```diff
@@ -0,0 +1,74 @@
+{
+    "@context": [
+        "https://w3id.org/ro/crate/1.1/context",
+        {
+            "organism_classification": "https://schema.org/taxonomicRange",
+            "BioChemEntity": "https://schema.org/BioChemEntity",
+            "obo": "http://purl.obolibrary.org/obo/",
+            "acquisiton_method": {
+                "@reverse": "https://schema.org/result",
+                "@type": "@id"
+            },
+            "biological_entity": "https://schema.org/about",
+            "preparation_method": "https://www.wikidata.org/wiki/Property:P1537"
+        }
+    ],
+    "@graph": [
+        {
+            "@type": "CreativeWork",
+            "@id": "ro-crate-metadata.json",
+            "conformsTo": {
+                "@id": "https://w3id.org/ro/crate/1.1"
+            },
+            "about": {
+                "@id": "./"
+            }
+        },
+        {
+            "@id": "./",
```

Comment:
I keep coming back to having this within the Zarr, though it's fair to consider what happens with an OME-TIFF. The most recent issue I had when discussing this issue on the [RO-Crate Regional Drop-In Calls](https://docs.google.com/document/d/1l0EmDVE4k6oEZSoMGjes8mMNrJboRlmZW-kvjaeVqvE/edit) is that one could imagine that someone might want to put "yet-another" RO-Crate at a level higher-up, then we would have:

 - new top RO-Crate
   - this RO-Crate
     - zarr
       - possibly more RO-Crates within

It just feels like we could save a level in the generic case.

Reply:
https://www.researchobject.org/ro-crate/specification/1.1/appendix/implementation-notes.html#combining-with-other-packaging-schemes may be relevant.